src/main/java/com/paneedah/mwc/MWC.java

```diff
@@ -46,6 +47,12 @@
 @Mod(modid = ID, name = NAME, version = VERSION, dependencies = "required-after:redcore@[0.4,);", guiFactory = "com.paneedah.weaponlib.config.ConfigGUIFactory", updateJSON = "https://raw.githubusercontent.com/Cubed-Development/Modern-Warfare-Cubed/master/update.json")
 public final class MWC {
 
+    static{
+        if(Loader.isModLoaded("groovyscript")){
+            com.paneedah.mwc.groovyscript.MWC.register();
+        }
+    }
+
```

Comment:
changed it to constructor on [1f43aa2](https://github.com/Cubed-Development/Modern-Warfare-Cubed/pull/303/commits/1f43aa26210a81a0627894a9a0dbaa87e0a1da56).
I was told that I need to call that method somewhere in main class or it could be ignored for being unused class.